packages/flutter/lib/src/material/date_picker_theme.dart

```diff
@@ -1158,6 +1174,12 @@ class _DatePickerDefaultsM3 extends DatePickerThemeData {
   @override
   TextStyle? get headerHelpStyle => _textTheme.labelLarge;
 
+  @override
+  Color? get headerTextColor => _colors.onSurfaceVariant;
```

Comment:
The default values in `_DatePickerDefaultsM3` are auto generated by running script. Could you help add the change in [here](https://github.com/flutter/flutter/blob/master/dev/tools/gen_defaults/lib/date_picker_template.dart#L80), and then run `dev/tools/gen_defaults/bin/gen_defaults.dart` in the root? The change added in date_picker_template.dart will then appear here.

Reply:
Same question here, should the default color be `_colors.onSurface.withOpacity(0.60)` to make sure it's backward compatible?